lib/wikisnakker.rb

```diff
@@ -142,11 +142,14 @@ def initialize(raw)
         [key, Sitelink.new(value)]
       end]
       raw['claims'].each do |property_id, claims|
-        property "#{property_id}s".to_sym do
-          claims.map { |c| Claim.new(c) }
+        define_property_method "#{property_id}s".to_sym do
+          # A claim's rank can be either preferred, normal or deprecated. We sort them by
+          # rank in reverse order because lexicographic ordering happens to work for the
+          # known ranks.
+          claims.map { |c| Claim.new(c) }.group_by(&:rank).sort_by { |r, _| r.reverse }.flat_map { |_, cs| cs }
```

Comment:
This won't work because the `r.reverse` means that the claims are actually being sorted by the reversed rank string, e.g. `lamron` instead of `normal`.